website/content/docs/operators/deployment/schedulers/kubernetes.md

```diff
@@ -0,0 +1,189 @@
+stored.
+
+* `heron.statemgr.zookeeper.is.initialize.tree` --- Specifies whether or not the State Manager can
+initialize the tree in Zookeeper. This should most often be set to true
+
+Most often with Heron deployments in a cluster environment, you are going to need to create a 
+tunneled connection to the cluster to submit and kill topology jars since communication with TMaster
+instances is needed. 
+
+To tunnel, you will need to add these extra configurations within your `statemgr.yaml` file.
+
+* `heron.statemgr.is.tunnel.needed` --- Flag for enabling the tunnel to a server within your 
+cluster. If you need the tunnel, which you most likely will, set this flag to `true`
+
+* `heron.statemgr.tunnel.connection.timeout.ms` --- The connection timeout in ms when trying to 
+connect to zk server
+
+* `heron.statemgr.tunnel.connection.retry.count` --- The count of retry to check whether has direct 
+access on zk server
+
+* `heron.statemgr.tunnel.retry.interval.ms` --- The interval in ms between two retry checking 
+whether has direct access on zk server
+
+* `heron.statemgr.tunnel.verify.count` --- The count of retry to verify whether seting up a tunnel 
+process
+
+* `heron.statemgr.tunnel.host` --- The host to SSH tunnel through in `<user>@<host>` format
+
+### Example State Manager Configuration
+
+```yaml
+# local state manager class for managing state in a persistent fashion
+heron.class.state.manager: com.twitter.heron.statemgr.zookeeper.curator.CuratorStateManager
+
+# local state manager connection string
+heron.statemgr.connection.string:  "<zk_ip>:2181"
+
+# path of the root address to store the state in a local file system
+heron.statemgr.root.path: "/heron"
+
+# create the zookeeper nodes, if they do not exist
+heron.statemgr.zookeeper.is.initialize.tree: True
+
+# timeout in ms to wait before considering zookeeper session is dead
+heron.statemgr.zookeeper.session.timeout.ms: 30000
+
+# timeout in ms to wait before considering zookeeper connection is dead
+heron.statemgr.zookeeper.connection.timeout.ms: 30000
+
+# timeout in ms to wait before considering zookeeper connection is dead
+heron.statemgr.zookeeper.retry.count: 10
+
+# duration of time to wait until the next retry
+heron.statemgr.zookeeper.retry.interval.ms: 10000
+
+
+heron.statemgr.is.tunnel.needed: True
+
+# The connection timeout in ms when trying to connect to zk server
+heron.statemgr.tunnel.connection.timeout.ms:    1000
+
+# The count of retry to check whether has direct access on zk server
+heron.statemgr.tunnel.connection.retry.count:   2
+
+# The interval in ms between two retry checking whether has direct access on zk server
+heron.statemgr.tunnel.retry.interval.ms:        1000
+
+# The count of retry to verify whether seting up a tunnel process
+heron.statemgr.tunnel.verify.count:             10
+
+# SSH tunnel host
+heron.statemgr.tunnel.host:              "<user>@<host>"
+```
+
+## Working with Topologies
+
```

Comment:
Could you please add some doc about sample heron submit/kill commands on kubernetics